### PR TITLE
Update execution of training file for NPM

### PR DESCRIPTION
**Pull request by lucky-suman**
As training file path is different for NPM from other ecosystems. So, need to update the path for training file.

### CODE REVIEW

File: rudra/deployments/emr_scripts/emr_config.py [hunk 1]
```diff
@@ -62,6 +63,12 @@ def get_config(self):
 
         execute_training_code = ['python3.6'.format(self.repo_dir),
                                  training_file, self.hyper_params]
+
+        if self.ecosystem=='npm':
+            training_file = "{}/{}".format(self.repo_dir, self.training_file_name)
```
Comment (miteshvp) [inline rudra/deployments/emr_scripts/emr_config.py]: training_file is already set earlier. No need to do it again

File: rudra/deployments/emr_scripts/emr_config.py [hunk 2]
```diff
@@ -62,6 +63,12 @@ def get_config(self):
 
         execute_training_code = ['python3.6'.format(self.repo_dir),
                                  training_file, self.hyper_params]
+
+        if self.ecosystem=='npm':
```
Comment (miteshvp) [inline rudra/deployments/emr_scripts/emr_config.py]: It's alway better to use `if-else` loop. Please do the same for `execute_training_code`

File: rudra/deployments/emr_scripts/emr_config.py (hunk 1 above)
Comment (lucky-suman) [reply rudra/deployments/emr_scripts/emr_config.py]: Ok!!

File: rudra/deployments/emr_scripts/emr_config.py (hunk 2 above)
Comment (lucky-suman) [reply rudra/deployments/emr_scripts/emr_config.py]: Ok, doing it :)

**Pull request merged**